plenum/common/test_network_setup.py

```diff
@@ -72,7 +73,7 @@ def bootstrapTestNodesCore(
             raise RuntimeError(
                 'nodeNum must be an int or set of ints') from exc
 
-        baseDir = cls.setup_base_dir(config)
+        baseDir = cls.setup_base_dir(config, envName)
```

Comment:
Do we take into account network_name in each script dealing with generation of keys? What about `init_bls_keys` and `init_plenum_keys`? What folder is used for generation of keys in tests?

Reply:
wait for Sergey's fix